gmso/formats/top.py

```diff
@@ -1,22 +1,23 @@
 import datetime
 
 import unyt as u
+import sympy
```

Comment:
Maybe i have missed it, but do we use sympy here?

Reply:
Removed that and the EngineIncompatibilityError import, which are both unused